src/App.js

```diff
@@ -1,29 +1,23 @@
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/calculator';
+
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <Calculator />
+        </header>
+      </div>
+    );
+  }
 }
```

Comment:
- [ ] You did a good job working on the **App** component. However, one of the requirements says we should use **functional components** for our components. So I would kindly recommend you update this component to a **functional component** instead. :ok_hand: 😊 suggesssion 

### Requirement :point_down: 
![Screenshot from 2023-05-02 20-09-34](https://user-images.githubusercontent.com/89228743/235775858-dfeb9846-c48c-4799-9855-6b907f6eac21.png)

```
const App = () => {
  return (
    <div className="App">
      <header className="App-header">
        <Calculator />
      </header>
    </div>
  );
};
```

Reply:
Hi @ridwanediallo ,

Thank you for your time and for your help. I updated my code with the functional component. 🙏